python/datamodel/nodesim/datamodel.py

```diff
@@ -106,6 +105,7 @@ def get_coordinates(self):
         elif hasattr(self, "_coordinates"):
             return self._coordinates
         else:
+            import numpy as np
```

Comment:
Being moved because I do not want numpy to be required when running the server for say car and pedestrian.